xarray/core/groupby.py

```diff
@@ -382,7 +382,15 @@ def fillna(self, value):
         Dataset.fillna
         DataArray.fillna
         """
-        return self._fillna(value)
+        def _yield_applied(this, other):
```

Comment:
> One solution is perhaps to add something into apply_groupby_ufunc to make sure the attrs are copied over, similar to what's in _copy_attrs_from in dataset.py/dataarray.py.

I like this -- it would be useful for other functions in `apply_ufunc`, too. For consistency, I would call the new argument `keep_attrs`.

Or if you don't want to bother with that, could just use `_copy_attrs_from` in this method:
```
out = ops.fillna(self, other)
out._copy_attrs_from(self._obj)
return out
```